.github/issue-pr-template.md

```diff
@@ -0,0 +1,6 @@
+Thank you for opening an Pull Request in our Repository.
```

Comment:
This template is different from the one discussed in https://app.asana.com/0/1204853730139328/1204778116780347/f

Feedback:
- "The issue has been forwarded..."  -> That seems incorrect since we are in the PR template, probably copy & paste?
- "**is** the Pull Request is not" -> probably a typo? do you mean "**if** the Pull Request..."
- IMO, the whole part about contribution guidelines seems unnecessary and more related to the scenario of opening an issue, could we just use something similar to option 1 in https://app.asana.com/0/1204853730139328/1204778116780347/f

I don't have a strong choice here, so feel free to pick the one you prefer.